src/main.rs

```diff
@@ -45,6 +45,19 @@ fn run() -> Result<(), failure::Error> {
         .author("ashley g williams <ashley666ashley@gmail.com>")
         .setting(AppSettings::ArgRequiredElseHelp)
         .setting(AppSettings::DeriveDisplayOrder)
+        .subcommand(
+            SubCommand::with_name("kv")
+                .about(&*format!(
+                    "{} Interact with your Workers KV Store",
```

Comment:
I would say "Interact with Workers KV"; this isn't wrong, but feels out of place to me. Can't really say why...

Reply:
I can take the verbiage from the zoneless dashboard: 
```Store application data in the Cloudflare network and access your key-value pairs from Workers.```

Reply:
this is also not a PR against master, so, everything in this feature branch is open to change before being released.

of course it is always nicer to iron things out early and not have to think about them any more.

Reply:
cool, let's not stress about it too much then :)